README.md

```diff
@@ -33,6 +42,18 @@ It can be found here: https://github.com/NedaRobatMeily/docker-image-java-11
 - Unblock the release step on **Buildkite**
 - On google play console, under releases section, add release note and publish the version
 
+## Project architecture
+
+This project it is implemented based on MVVM pattern and repository pattern.
```

Comment:
Could you explain more about MVVM and the layers?
You could refer to https://github.com/ROKT/rokt-demo-ios as an example. It is a good time to visit the MVVM structure